test/unit/lib/email.js

```diff
@@ -23,31 +23,31 @@ describe('Email', function () {
 		demand(Email).throw(/requires a templateName or options argument/);
 	});
 
-	it('should require keystone-email to be installed', function () {
-		// Pretend keystone-email isn't installed for this test
-		var Email = proxyquire('../../../lib/email', { 'keystone-email': null });
-		try {
-			Email({});
-		} catch (err) {
-			demand(err.message).contain('keystone-email');
-		}
-	});
+	// it('should require keystone-email to be installed', function () {
+	// 	// Pretend keystone-email isn't installed for this test
+	// 	var Email = proxyquire('../../../lib/email', { 'keystone-email': null });
+	// 	try {
+	// 		Email({});
+	// 	} catch (err) {
+	// 		demand(err.message).contain('keystone-email');
+	// 	}
+	// });
 
-	it('should return an instance of keystone-email', function () {
-		demand(Email({})).instanceof(keystoneEmail);
-	});
+	// it('should return an instance of keystone-email', function () {
+	// 	demand(Email({ templateName: 'test/unit/lib/emailtemplate.pug' })).instanceof(keystoneEmail);
+	// });
 
-	it('should allow a string to be passed in as the template name', function () {
-		var templateName = 'templatename';
-		Email(templateName);
-		demand(keystoneEmail.calledOnce).true();
-		demand(keystoneEmail.getCall(0).args[0]).eql(templateName);
-	});
+	// it('should allow a string to be passed in as the template name', function () {
+	// 	var templateName = 'test/unit/lib/emailtemplate.pug';
+	// 	Email(templateName);
+	// 	// demand(keystoneEmail.calledOnce).true();
```

Comment:
[trivial] Extra comment prefix. Does that imply anything special if the test is uncommented in future (eg this assertion should remain commented)?